app/views/warehouse_reports/outflow/_filter.haml

```diff
@@ -24,8 +24,11 @@
     .row
       .col-8
```

Comment:
Widen this column since it is covering the entire row?

Reply:
maybe, but the look weird when they are too wide.